lib/models/project.js

```diff
@@ -750,6 +749,7 @@ function ensureInstrumentation(cli, ui) {
     return cli.instrumentation;
   }
 
+  let Instrumentation = require('./instrumentation');
```

Comment:
I needed to move the Instrumentation require inline otherwise it blew up on init due to circular dependencies between Instrumentation => getConfig => Project => Instrumentation